packages/compass-smoke-tests/src/releases.ts

```diff
@@ -0,0 +1,115 @@
+import { downloadFile } from './downloads';
+import { type PackageKind } from './packages';
+
+type Arch = 'arm64' | 'x64';
+
+type PlatformShortName =
+  | 'darwin-arm64'
+  | 'darwin-x64'
+  | 'windows'
+  | 'linux_deb'
+  | 'linux_rpm';
+
+function getPlatformShortName(
```

Comment:
I'd like a doc string on this function - especially the return value of the kinds confuses me.

Reply:
What do you want the doc string to say?